pkgs/by-name/ce/censor/package.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  fetchFromGitea,
+  python3Packages,
+  wrapGAppsHook4,
+  gobject-introspection,
+  libadwaita,
+  meson,
+  ninja,
+  pkg-config,
+  desktop-file-utils,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "censor";
+  version = "0.3.0";
+  pyproject = false;
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "censor";
+    repo = "Censor";
+    rev = "v${version}";
+    hash = "sha256-16Cy9yNOLvdVZ234kTB8fa585eI4f7yVnbMoPcHtGHk=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    gobject-introspection
+    wrapGAppsHook4
+    desktop-file-utils
+  ];
+
+  buildInputs = [
+    libadwaita
+  ];
+
+  dependencies = with python3Packages; [
+    pygobject3
+    pymupdf
+  ];
+
+  dontWrapGApps = true;
+
+  preFixup = ''
+    makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
+  '';
```

Comment:
I could be wrong, but these lines (45-49) may be redundant? I'm not sure. Try removing both of them and see if it works...

I think the wrapGAppsHook4 should take care of this